test_config/test_syntax.yml

```diff
@@ -0,0 +1,36 @@
+description: <Job Description>    # Optional
+service: <GA4GH Service Name>   # enum - [TES]
```

Comment:
Probably best to remove GA4GH here (and possibly elsewhere). I don't see why the runner would not work on OpenAPI specs that are not developed by GA4GH